### PR TITLE
Add SessionFeedback Table

### PR DESCRIPTION
**Pull request by futabooo**
## Issue
- related #40

## Overview (Required)
- Add SessionFeedback Table
    - need to manage whether submitted feedback for UI
    - I think, there is needs to save session feedback before submit

## Screenshot
ui no change

### CODE REVIEW

File: common/src/main/kotlin/io/github/droidkaigi/confsched2018/model/SessionFeedback.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+data class SessionFeedback(
+        val sessionId: String,
+        val sessionTitle: String,
+        val totalEvaluation: Int,
+        val relevancy: Int,
+        val asExpected: Int,
+        val difficulty: Int,
+        val knowledgeable: Int,
+        val comment: String,
+        var submitted: Boolean
```
Comment (konifar) [inline common/src/main/kotlin/io/github/droidkaigi/confsched2018/model/SessionFeedback.kt]: @mhidaka The session feedback items are fixed? 🙋

**Comment by konifar** [comment]
Thanks!
LGTM! But I'm asking to DroidKaigi staff to check the items just in case.

File: app/src/main/java/io/github/droidkaigi/confsched2018/data/db/dao/SessionFeedbackDao.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+@Dao abstract class SessionFeedbackDao {
+    @Query("SELECT * FROM session_feedback")
+    abstract fun getAllSessionFeedback(): Flowable<List<SessionFeedbackEntity>>
+
+    @Query("DELETE FROM session_feedback WHERE session_id = :sessionId")
+    abstract fun delete(sessionId: String)
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    abstract fun insert(sessionFeedback: SessionFeedbackEntity)
```
Comment (jmatsu) [inline app/src/main/java/io/github/droidkaigi/confsched2018/data/db/dao/SessionFeedbackDao.kt]: upsert is preferred in SQL world.

File: app/src/main/java/io/github/droidkaigi/confsched2018/data/db/dao/SessionFeedbackDao.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    @Transaction open fun clearAndInsert(sessionFeedback: SessionFeedbackEntity) {
```
Comment (jmatsu) [inline app/src/main/java/io/github/droidkaigi/confsched2018/data/db/dao/SessionFeedbackDao.kt]: If you don't need to create new row id, then it's enough to use `insert`.

Comment (futabooo) [reply app/src/main/java/io/github/droidkaigi/confsched2018/data/db/dao/SessionFeedbackDao.kt]: yes I want to update row.
If I misunderstood the intention of the question, sorry.

**Comment by futabooo** [comment]
thank you for rebiew!fix comment!

**Comment by konifar** [comment]
Thanks! just in case, I'll wait @jmatsu 's review again.

**Comment by konifar** [comment]
Okay! Let's merge this! 😄 
Thanks @futabooo

**Pull request merged**